### PR TITLE
Recipe model

### PR DESCRIPTION
**Pull request by tsop14**
- Separates ui and common projects from root.
- Adds model for auto-parsed recipes.

(TeamCity build step set sbt to an old version, which failed to build this branch. This line was removed.)

### CODE REVIEW

File: common/src/main/scala/com/gu/recipeasy/models/Recipe.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.gu.recipeasy.models
+
+import java.time.OffsetDateTime
+
+case class Recipe(
+  id: String,
```
Comment (cb372) [inline common/src/main/scala/com/gu/recipeasy/models/Recipe.scala]: title is missing

File: common/src/main/scala/com/gu/recipeasy/models/Recipe.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  body: String,
+  serves: Option[Serves],
+  ingredients: Seq[IngredientList],
+  articleId: String,
+  credit: Option[String],
+  publicationDate: OffsetDateTime,
+  status: Status,
+  steps: Option[List[String]]
+)
```
Comment (cb372) [inline common/src/main/scala/com/gu/recipeasy/models/Recipe.scala]: Also we should add `imageUrl: Option[String]`. We can extend the parser to try to find an image for each recipe, as we discussed.

**Comment by cb372** [comment]
👍  apart from a couple of missing fields

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**